pkg/operator/customroute/manifests/route.yaml

```diff
@@ -0,0 +1,21 @@
+# emulates server-side defaulting as in https://github.com/openshift/openshift-apiserver/blob/master/pkg/route/apis/route/configv1listers/defaults.go
```

Comment:
this does not belong to library-go but to the CAO

Reply:
Changed to a function call, see https://github.com/openshift/cluster-authentication-operator/pull/478/files#diff-0d623dfd885adb20f991bda4c2453aebd732ca6dbb4d1d4be6e79805c3b48de6R445